packages/server/graphql/mutations/helpers/inviteToTeamHelper.ts

```diff
@@ -0,0 +1,225 @@
+    removedSuggestedActionId = await removeSuggestedAction(viewerId, 'inviteYourTeam')
+  }
+  // insert notification records
+  const notificationsToInsert = [] as NotificationTeamInvitation[]
+  teamInvitationsToInsert.forEach((invitation) => {
+    const user = users.find((user) => user.email === invitation.email)
+    if (user) {
+      notificationsToInsert.push(
+        new NotificationTeamInvitation({
+          userId: user.id,
+          invitationId: invitation.id,
+          teamId
+        })
+      )
+    }
+  })
+  if (notificationsToInsert.length > 0) {
+    await r.table('Notification').insert(notificationsToInsert).run()
+  }
+
+  const bestMeeting = await getBestInvitationMeeting(teamId, meetingId ?? undefined, dataLoader)
+
+  // send emails
+  const searchParams = {
+    utm_source: 'invite email',
+    utm_medium: 'email',
+    utm_campaign: 'invitations'
+  }
+  const options = {searchParams}
+  const emailResults = await Promise.all(
+    teamInvitationsToInsert.map((invitation) => {
+      const user = users.find((user) => user.email === invitation.email)
+      const {html, subject, body} = teamInviteEmailCreator({
+        appOrigin,
+        inviteLink: makeAppURL(appOrigin, `team-invitation/${invitation.token}`, options),
+        inviteeName: user ? user.preferredName : '',
+        inviteeEmail: invitation.email,
+        inviterName: inviter.preferredName,
+        inviterEmail: inviter.email,
+        teamName,
+        meeting: bestMeeting,
+        corsOptions: EMAIL_CORS_OPTIONS
+      })
+      return getMailManager().sendEmail({
+        to: invitation.email,
+        html,
+        subject,
+        body,
+        tags: [
+          'type:teamInvitation',
+          `tier:${tier}`,
+          `team:${teamName}:${orgName}:${teamId}:${orgId}`
+        ]
+      })
+    })
+  )
+
+  const parabolUserEmails = users.map(({email}) => email)
+  const inviteTo = meetingId ? 'meeting' : 'team'
+  newAllowedInvitees.forEach(async (inviteeEmail, idx) => {
+    const isInviteeParabolUser = parabolUserEmails.includes(inviteeEmail)
+    const success = !!emailResults[idx]
+    analytics.inviteEmailSent(
+      viewerId,
+      teamId,
+      inviteeEmail,
+      isInviteeParabolUser,
+      inviteTo,
+      success
+    )
+  })
+  const successfulInvitees = newAllowedInvitees.filter((_email, idx) => emailResults[idx])
+  const data = {
+    removedSuggestedActionId,
+    teamId,
+    invitees: successfulInvitees
+  }
+
+  // Tell each invitee
+  notificationsToInsert.forEach((notification) => {
+    const {userId, id: teamInvitationNotificationId} = notification
+    const subscriberData = {
+      ...data,
+      teamInvitationNotificationId
+    }
+    publish(
+      SubscriptionChannel.NOTIFICATION,
+      userId,
+      'InviteToTeamPayload',
+      subscriberData,
+      subOptions
+    )
+  })
+
+  return {
+    removedSuggestedActionId,
+    invitees: successfulInvitees,
+    notificationsToInsert
+  }
```

Comment:
+1 I see this only used in one place `inviteToTeam` and there you need to modify it to match the local `data` variable here, so why not just return that here?